deploy-precise-code-intel-bundle-manager.sh

```diff
@@ -8,6 +8,8 @@ source ./replicas.sh
 # Ports exposed to other Sourcegraph services: 3187/TCP
 # Ports exposed to the public internet: none
 #
+VOLUME="$HOME/sourcegraph-docker/lsif-server-disk"
+./ensure-volume.sh $VOLUME 999
```

Comment:
Why has this changed?

Reply:
without this, by default I am getting the following errors during the tests:
```
    pure-docker-test-local: TEST FAILURE: expected 23 containers running, found 22
```

```
root@vagrant:~# docker logs -f f35c284a2fef
2020/08/24 23:36:10 failed to prepare directories: mkdir /lsif-storage/uploads: permission denied
```

Reply:
I"m not sure why this doesn't have a the volume assigned to a user, given every other service we have that uses a volume runs this script. I've changed to the sourcegraph user as per the other services that run under that account. 

Related PR [here](https://github.com/sourcegraph/sourcegraph/pull/13383) should this get merged.